src/dsp_tools/utils/stack_handling.py

```diff
@@ -91,7 +93,9 @@ def start_stack(
         ("knora-ontologies/standoff-data.ttl", "http://www.knora.org/data/standoff"),
         ("knora-ontologies/salsah-gui.ttl", "http://www.knora.org/ontology/salsah-gui"),
         ("test_data/all_data/admin-data-minimal.ttl", "http://www.knora.org/data/admin"),
```

Comment:
as mentioned in chat, this should not be minimal, so that the tests pass.

```suggestion
        ("test_data/all_data/admin-data.ttl", "http://www.knora.org/data/admin"),
```